chore: add border radius on components (#2543)
WTTJ/welcome-ui: website/build-app/pages/foundations/theming/basics.mdx

<div data-theme="space"></div>

## Border radius

> By default, we have a border radius on components. You can remove it by set to 0 the radii borders. You can also configure the width of the border.

Reviewer: ```suggestion
By default, we set a border radius on components. Set the radii borders  to 0 if you don't like this default. You can also configure the border width.
```

Author: I need to chatgpt all our texts :D
